.gitignore

```diff
@@ -124,4 +124,7 @@ ENV/
 ddtrace/_version.py
 
 # Benchmarks
-artifacts/
\ No newline at end of file
+artifacts/
+
+# Vale
+.github/styles/*
```

Comment:
Looks like there's a checked-in file in this directory

Reply:
Added negation to include vocabulary entries